biography.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Biograph-eh</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+
+  <body>
+
+  <h1>A Little Aboot Me</h1>
+
+  <p class=info-paragraphs>
+    Coincidentally born on Christmas Day in 1971, Justin Trudeau was placed on Earth to be humanity's literal lord and savior. Having been in the shadows of his father, Canadian Prime Minister Pierre Trudeau, it's no wonder that Justin himself would step into that limelight. Originally, Justin tried steering away from the political field. He studied literature at McGill University, and education at the University of British Columbia. Eventually, politics called his name and he chaired for the Liberal Party's youth task force in 2006. In 2008 he won a seat in the Canadian Parliament. Later in 2013, he became the Liberal Party's leader (so much for staying away from politics, right?) Finally, in 2015 J-Trudes proved he was the Supreme when he was elected Prime Minister of Canada. Today, he's proven to be one of Canada's most open-minded leaders, and the only force that can defeat <span class="voldemort">Lord Voldemort</span> Donald Trump.
+
+    <img src="http://pbs.twimg.com/media/CSF6BMsWEAAaqoA.png" id="bio-pics1">
```

Comment:
Refrain from using id's for styling.  You can only have one per page, the specificity is hard to over come (https://css-tricks.com/specifics-on-css-specificity/) which will likely cause issues down the road, and you lose the benefit of CSS since you will need repeat styles for every element that could have just been styled once using the same class.  While not evil, it is not best practice and should only be used in certain edge cases (https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/).